### PR TITLE
Remove Study-related info from lighter/apis/poc/setup.py/requirements…

### PR DESCRIPTION
**Pull request by IsaacYangSLA**
….txt

Need PR#532.

### CODE REVIEW

**Comment by IsaacYangSLA** [comment]
@nvkevlu , sorry that I don't know how to update the ui html file.

**Comment by nvkevlu** [comment]
> @nvkevlu , sorry that I don't know how to update the ui html file.

I can update that

File: nvflare/apis/impl/job_def_manager.py [hunk 1]
```diff
@@ -63,25 +62,14 @@ def __init__(self, reviewer_name, fl_ctx: FLContext):
...
-                return True
-            else:
-                self.result.append(job_from_meta(meta))
-        return True
+        approvals = meta.get(JobMetaKey.APPROVALS)
+        if approvals and self.reviewer_name in approvals:
+            # already reviewed
+            return True
+        else:
+            self.result.append(job_from_meta(meta))
```
Comment (YuanTingHsieh) [inline nvflare/apis/impl/job_def_manager.py:72]: missing a `return True` in the end

**Review by YuanTingHsieh — CHANGES_REQUESTED** [review]
LGTM, can you address the comment and rebase the code thanks!

File: nvflare/apis/impl/job_def_manager.py (hunk 1 above)
Comment (IsaacYangSLA) [reply nvflare/apis/impl/job_def_manager.py:69]: Added.  Thanks.

**Review by YuanTingHsieh — APPROVED** [review]
Thanks, LGTM

**Pull request merged**